ramls/item.json

```diff
@@ -231,7 +231,8 @@
             "On order",
             "Paged",
             "Declared lost",
-            "Order closed"
+            "Order closed",
+            "Claim returned"
```

Comment:
Are there any tests for item status in this module?

Reply:
I don't think it make sense, it is a pass through operation and RMB is responsible of it.

Reply:
What do you mean by it is a passthrough?

Does that mean we don't test at least some of the various statuses in the API tests?

Reply:
I meant that the only think that validates the status is RMB.  

No, we have a test for `Available` and `Checked out` statuses.

Reply:
Hmm, ok, that seems a bit odd to me. Now that the list is constrained, this should probably be tested.

Reply:
@marcjohnson-kint, I have added a test to verify that item can be created with all the allowed status values.